### DISCUSSION TITLE
Is AWS code artifact support to renovate bot.

### DISCUSSION THREAD

**Discussion by SORABH13** [renovatebot/renovate#19663] [Request Help]
Hi Team,
Please help me to understand my scenario. 

Is AWS code artifact support to renovate bot??

Like I have renovate configure in my repo but I want renovate first go inside my aws code artifactory and check version and accordingly generate dependency lookups warning. 

If support AWS code artifact for NPM. Please suggest how can I configure aws code artifact scan in my renovate project?

**Comment by viceice** [comment]
like any other npm registry? Checkout our docs.

**Reply by HonkingGoose** [reply]
Related issue:

- https://github.com/renovatebot/renovate/issues/13833
